### DISCUSSION TITLE
Question: How to display clickable links on Desktop?

### DISCUSSION THREAD

**Discussion by murty2** [DustinBrett/daedalOS#441] [Q&A]
I want to show links on Desktop. When a user clicks on the link, it should open the link in another browser window, not inside daedalOS

How to display such clickable links on Desktop?

**Comment by DustinBrett** [comment]
To do that you would make a `.url` file with a `BaseURL` to `ExternalURL` and a `URL` to the url you want to open. You can also add a `IconFile` entry to give it an icon.

```
[InternetShortcut]
BaseURL=ExternalURL
URL=https://www.google.com/
IconFile=...
```

If you put this in the public folder then `scripts\cacheShortcuts.js` can make a cached entry of it so it doesn't need to request a real file.

**Comment by murty2** [comment]
Is it possible show actual clickable word on the Desktop, as in just text instead of an Icon?

**Reply by DustinBrett** [reply]
That isn't something I have created so I would think no, you'd need to code such an element/component. You could use a transparent icon.
